src/screens/Main/Dashboard/Dashboard.jsx

```diff
@@ -1,11 +1,12 @@
 import { Action, Input, Screen, ScrollView } from '@satoshi-ltd/nano-design';
 import PropTypes from 'prop-types';
 import React, { useEffect, useState } from 'react';
+import { SectionList } from 'react-native';
```

Comment:
this one should be part of nano-design.. the general idea is money don't use any `react-native` component by itself.

Reply:
Done